assets/blocks/course-list-block/course-list.scss

```diff
@@ -49,3 +61,49 @@ $block: '.wp-block-sensei-lms-course-list';
 		padding: 1.25em;
 	}
 }
+
+.featured-badge {
+	display: none;
+}
+
+
+.featured-image-wrapper {
+	position: relative;
+	.featured-badge {
+		display: block;
+		position: absolute;
+		top: 10px;
+		left: 10px;
+		background-color: #1d2327;
+		padding: 3px 6px;
+		text-decoration: none;
+		border-radius: 2px;
+		white-space: nowrap;
+		z-index: 1000;
+	}
+	.featured-badge-edit {
+		height: auto !important;
+	}
+}
+
+
+.featured-category-wrapper {
+	display: flex;
+	flex-direction: row;
+
+	.featured-badge {
+		display: block;
```

Comment:
Since some of the CSS here is shared with the CSS above, a [mixin](https://sass-lang.com/documentation/at-rules/mixin) could be useful for the reusable bits. You can see an example [here](https://github.com/Automattic/sensei/blob/trunk/assets/blocks/learner-courses-block/learner-courses-mixins.scss).

Reply:
Fixed here 93244132c78e2d64744a877abfefdeb02fc1ed53